code/game/machinery/medipen_refiller.dm

```diff
@@ -0,0 +1,81 @@
+/obj/machinery/medipen_refiller
+	name = "Medipen Refiller"
+	desc = "A machine that refills used medipens with chemicals, it needs them first."
```

Comment:
Change to “A machine that refills used medipens.”
Saying what it can be filled with is redundant. It’s a medipen, of course it will have meds in it. As for